canal/canal.go

```diff
@@ -380,6 +380,18 @@ func (c *Canal) ClearTableCache(db []byte, table []byte) {
 	c.tableLock.Unlock()
 }
 
+// SetTableCache sets table cache value for the given key
```

Comment:
"for the given key" maybe not as straightforward as "for the given table"

Reply:
Changed it to be "for the given table". Thanks!